rotkehlchen/data_migrations/migrations/migration_22.py

```diff
@@ -39,4 +39,4 @@ def _remove_legacy_coinbase_keys(rotki: 'Rotkehlchen') -> None:
                 [(cred.name, cred.location.serialize_for_db()) for cred in creds_to_remove],
             )
 
-    perform_userdb_migration_steps(rotki, progress_handler, should_vacuum=True)
+    perform_userdb_migration_steps(rotki, progress_handler)
```

Comment:
nitpick: Can you just set it to False please? It's quite possible we will forget its existence by the time of the next migraiton